src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/PublicDataDbContextModelSnapshot.cs

```diff
@@ -668,6 +668,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .ValueGeneratedOnAdd()
                         .HasColumnType("uuid");
 
+                    b.Property<DateTimeOffset?>("Activates")
+                        .HasColumnType("timestamp with time zone");
+
```

Comment:
I can't see the generated migration code associated with this change when you created the migration. I think you might have forgot to add them to the commit.